src/transaction-flow/TransactionFlowProvider.tsx

```diff
@@ -174,6 +174,7 @@ export const TransactionFlowProvider = ({ children }: { children: ReactNode }) =
           key,
           payload: flow,
         })) as CreateTransactionFlow,
+      dispatch,
```

Comment:
ideally dispatch isnt exposed from the context so that we are strict on what reducer actions are called

Reply:
why is it bad? if a reducer action shouldn't be called then it probably shouldn't be there?

Reply:
looks like you made this change but just to clarify, i see `dispatch` itself more as an internal handler for the provider which doesn't do the same checks/preprocessing that the externally exposed functions do, so it's just preventing any unintentional usage of any given reducer action.